### ISSUE TITLE
disabled IconButton has no accessible name

### ISSUE THREAD

**Issue by straker** [dequelabs/cauldron#416]
```html
<IconButton
  disabled
  icon="pencil"
  label="Edit"
  variant="secondary"
/>
```

Fails axe clean. Also JAWS / Chrome reading line by line can focus disabled buttons, which announces ""Unlabeled 0 button, unavailable."

**Comment by harrisonwillettbetterworks** [comment]
Disabled buttons can not receive focus. In the case of an Edit button you may want to use an attribute like aria-pressed ="true" instead so designate that the button still exists but is in the editing state.

**Comment by dequejosie** [comment]
Duplicate of https://github.com/dequelabs/cauldron/issues/309; I am closing this.
